github-clone.el

```diff
@@ -180,9 +180,11 @@ DIRECTORY.  Then it prompts to fork the repository and add a
 remote named after the github username to the fork."
   (interactive
    (list (read-from-minibuffer "Url or User/Repo: ")
-         (read-directory-name "Directory: " nil default-directory t)))
+         (read-directory-name "Directory: " nil default-directory)))
   (let* ((name (github-clone-repo-name user-repo-url))
          (repo (github-clone-info (car name) (cdr name))))
+    (unless (file-directory-p directory)
```

Comment:
this check is actually unnecessary, since `make-directory` is idempotent and will do nothing when asked to "make" an existing directory, but this looks a bit more readable to me personally